uk.co.compucorp.civicrm.hrleaveandabsences/js/src/leave-absences/leave-type-wizard/components/leave-type-wizard.component.js

```diff
@@ -302,9 +354,34 @@ define([
     }
 
     /**
-     * Saves the whole wizard
+     * Saves leave type by sending an API call to the backend
+     * with all appropriate parameters.
      */
     function save () {
+      var params = _.chain(vm.fieldsIndexed)
+        .keyBy('name')
+        .mapValues('value')
+        .value();
+
+      vm.loading = true;
+
+      preProcessParams(params);
+      AbsenceType.save(params)
+        .then(navigateToLeaveTypesList)
```

Comment:
no nice success message after saving the absence type?

Reply:
Good call, but there was no requirement for that.